src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -38,6 +50,10 @@ class ThumbnailsPanel extends React.PureComponent {
     core.addEventListener('pageNumberUpdated', this.onPageNumberUpdated);
     core.addEventListener('annotationHidden', this.onAnnotationChanged);
     window.addEventListener('resize', this.onWindowResize);
+
+    if (this.thumbnailsPanel && this.thumbnailsPanel.current) {
+      this.thumbnailsPanel.current.addEventListener('drop', this.onDrop);
```

Comment:
Hmm maybe I'm missing something, but can we just pass an onDrop prop to the thumbnails panel div? If we can do that we don't need to ref it.

Reply:
I updated it to use the "onDrop" prop, I can't remember why I was doing it like this